packages/cloud_firestore/cloud_firestore_web/lib/query_web.dart

```diff
@@ -62,16 +76,24 @@ class QueryWeb implements Query {
         this._firestore,
         this._path,
         webQuery != null ? webQuery.limit(length) : null,
+        orderByKeys: _orderByKeys,
         isCollectionGroup: _isCollectionGroup,
       );
 
   @override
-  Query orderBy(field, {bool descending = false}) => QueryWeb(
-        this._firestore,
-        this._path,
-        webQuery.orderBy(field, descending ? "desc" : "asc"),
-        isCollectionGroup: _isCollectionGroup,
-      );
+  Query orderBy(field, {bool descending = false}) {
+    dynamic usableField = field;
```

Comment:
`field` can be anything.. `usableField` has been type checked to avoid passing types that `jsify`  doesn't like.

It's mainly for catching `FieldPath` and passing it as `web.FieldPath`

Reply:
Ahhh ok, makes sense. Thanks for the explanation!